beacon-chain/p2p/pubsub.go

```diff
@@ -147,9 +147,34 @@ func (s *Service) pubsubOptions() []pubsub.Option {
 		pubsub.WithGossipSubParams(pubsubGossipParam()),
 		pubsub.WithRawTracer(gossipTracer{host: s.host}),
 	}
+
+	if len(s.cfg.StaticPeers) > 0 {
+		s.addDirectPeersOption(psOpts)
+	}
+
 	return psOpts
 }
 
+// addDirectPeersOption parses the static peers from the service configuration and adds
+// the direct peers option to the pubsub options.
+func (s *Service) addDirectPeersOption(psOpts []pubsub.Option) {
+	addrs, err := PeersFromStringAddrs(s.cfg.StaticPeers)
+
+	if err != nil {
+		log.WithError(err).Error("Could not convert static peers raw ENRs into multiaddresses")
+		return
+	}
```

Comment:
If 0 valid `addrs` are returned from `PeersFromStringAddrs`, we should log an error/warning and exit early

Reply:
I've just added a warning log! Thank you